Content.Shared/Mobs/Systems/MobStateSystem.Subscribers.cs

```diff
@@ -28,7 +28,7 @@ private void SubscribeEvents()
         SubscribeLocalEvent<MobStateComponent, ChangeDirectionAttemptEvent>(CheckAct);
         SubscribeLocalEvent<MobStateComponent, UseAttemptEvent>(CheckAct);
         SubscribeLocalEvent<MobStateComponent, AttackAttemptEvent>(CheckAct);
-        SubscribeLocalEvent<MobStateComponent, InteractionAttemptEvent>(CheckAct);
+        SubscribeLocalEvent<MobStateComponent, ConsciousAttemptEvent>(CheckAct);
```

Comment:
Can you also add a check for sleeping players?

Reply:
Not sure if I misunderstood you, in discord you said the consciousness check should check if the user is asleep. It does that as the sleepsystem subscribes to it so this should be resolved?

Reply:
nik is right this is already checked for sleeping